rails6/en/chapter05-user-products.adoc

```diff
@@ -561,7 +561,7 @@ $ rake test
 
 === Destroying products
 
-Our last stop for the products endpoints will be the `destroy` action. You might now imagine how this would look like. The strategy in here will be pretty similar to the `create` and `update` actions: we'll get the logged user with JWT token and then fetch the product from the `user.products` association and finally destroy it, returning a `204` code.
+Our last stop for the product endpoints will be the `destroy` action. You might now imagine how this would look like. The strategy in here will be pretty similar to the `create` and `update` actions: we'll get the logged user with JWT token and then fetch the product from the `user.products` association and finally destroy it, returning a `204` code.
```

Comment:
What do you think about ?

```suggestion
Our last stop for products endpoints will be the `destroy` action. You might now imagine how this would look like. The strategy in here will be pretty similar to the `create` and `update` actions: we'll get the logged user with JWT token and then fetch the product from the `user.products` association and finally destroy it, returning a `204` code.
```

Reply:
We need "the" here.